app/components/DayPickerInput/index.tsx

```diff
@@ -3,6 +3,10 @@ import BaseDayPickerInput from 'react-day-picker/DayPickerInput'
 import styles from 'react-day-picker/lib/style.css'
 import customStyles from './styles.css'
 
+interface IDayPickerInputProps extends DayPickerInputProps {
+  onDayChange: (selectedDay: Date) => void
+}
```

Comment:
The project is using `type` instead of `interface`.
And for this solution, the import could be `import type { DayPickerInputProps as ReactDayPickerInputProps } from 'react-day-picker'`

```suggestion
type DayPickerInputProps = {
  onDayChange: (selectedDay: Date) => void
} & ReactDayPickerInputProps
```